client/signup/steps/site-type/index.jsx

```diff
@@ -129,10 +129,9 @@ export default connect(
 			let nextFlowName = ownProps.flowName;
 			if ( siteTypeValue === getSiteTypePropertyValue( 'id', 'store', 'slug' ) ) {
 				nextFlowName = 'ecommerce';
-			} else if ( 'ecommerce' === ownProps.flowName && ownProps.previousFlowName ) {
-				nextFlowName = ownProps.previousFlowName;
+			} else if ( siteTypeValue === getSiteTypePropertyValue( 'id', 'business', 'slug' ) ) {
```

Comment:
I just realized: this is the line that plugs it all in. 😄 Merging will effectively 'switch on' site style/preview in the flow for all users in the `onboarding` flow. 

Holding off on merging this PR until the final site preview designs have been implemented 

Since the preview styles are not yet finalized, we could revert this block and enable it only for `onboarding-dev` if we want to merge now, e.g.,

```
if ( siteTypeValue === getSiteTypePropertyValue( 'id', 'store', 'slug' ) ) {
				nextFlowName = 'ecommerce';
			} else if ( 'ecommerce' === ownProps.flowName && ownProps.previousFlowName ) {
				nextFlowName = ownProps.previousFlowName;
			} else if (  ( siteTypeValue === getSiteTypePropertyValue( 'id', 'business', 'slug' ) && 'onboarding-dev' === ownProps.flowName )  ) {
                ....
```
It might be useful to have to the updated flow config in dev to work with.

Or just put this PR on ice until the site preview is finalized. 

@michaeldcain

Reply:
> Since the preview styles are not yet finalized, we could revert this block and enable it only for onboarding-dev if we want to merge now

@ramonjd: sorry I missed the ping. Can we go ahead and launch this to just the onboarding-dev flow?